code/modules/busy_space/air_traffic.dm

```diff
@@ -52,17 +52,16 @@ GLOBAL_DATUM_INIT(lore_atc, /datum/lore/atc_controller, new)
 	ASSERT(message)
 	GLOB.global_announcer.autosay("[message]", sender ? sender : "[GLOB.using_map.dock_name] Control")
 
-/datum/lore/atc_controller/proc/reroute_traffic(yes = TRUE)
-	if(yes)
-		if(!squelched)
-			msg("Re-routing traffic and fleet patterns around [GLOB.using_map.station_name].")
+/datum/lore/atc_controller/proc/reroute_traffic()
```

Comment:
just an Idea, rename this to `toggle_broadcast`

Reply:
Oh yeah, given the change in the flavor text surrounding it this is fair.

Reply:
should be done in a series of webedits xd